project.clj

```diff
@@ -22,10 +22,6 @@
     heap-size-from-profile-clj
     default-heap-size))
 
-(def figwheel-version "0.3.7")
-(def cljsbuild-version "1.1.7")
-(def clojurescript-version "1.10.238")
```

Comment:
Oh wow, I'm impressed this is still in here. I feel like we've done several rounds of cleaning up the vestiges of that :P